lib/hu_board/services/slack.rb

```diff
@@ -29,7 +29,7 @@ def github_url
     def transform_moved(mash)
       {
         username: "#{mash.meta.user.login} via HuBoard",
-        icon_url: "https://avatars.githubusercontent.com/u/#{mash.meta.user.id}?s=64",
+        icon_url: "#{mash.payload.issue.user.avatar_url}s=64",
```

Comment:
Actually, let's just drop the size and let Slack browser/client scale appropriately.

<img width="443" alt="firefox_2016-03-15_18-18-23" src="https://cloud.githubusercontent.com/assets/133987/13797272/57ad6980-eada-11e5-911c-16031aaaf3ab.png">